test_msgs/include/test_msgs/action/fibonacci.h

```diff
@@ -0,0 +1,24 @@
+// generated from rosidl_generator_c/resource/msg__type_support.h.em
+// generated code does not contain a copyright notice
+
+#ifndef TEST_MSGS__ACTION__FIBONACCI_H_
+#define TEST_MSGS__ACTION__FIBONACCI_H_
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include <action_msgs/msg/goal_info.h>
+#include <action_msgs/msg/goal_status_array.h>
+#include <action_msgs/srv/cancel_goal.h>
+#include <test_msgs/action/fibonacci__feedback.h>
+#include <test_msgs/action/fibonacci__goal.h>
+#include <test_msgs/action/fibonacci__result.h>
+#include "test_msgs/action/fibonacci__type_support.h"
+
+#ifdef __cplusplus
+}
+#endif
+
+#define TEST_MSGS__ACTION__FIBONACCI_H_
```

Comment:
@sloretz `#endif  // TEST_MSGS__ACTION__FIBONACCI_H_ ` ?

Reply:
Fixed in f99efcfaa4e06d38569cafef87a46b33bdf39457